netci.groovy

```diff
@@ -1512,6 +1526,10 @@ def static addNonPRTriggers(def job, def branch, def isPR, def architecture, def
             if (os in bidailyCrossList) {
                 break
             }
+            if ((os == 'Ubuntu') && (architecture == 'arm') && !isCoreFxScenario(scenario)) {
+                // Linux arm32 duplicated by AzDO
```

Comment:
not all of them are duplicated by AzDO but I think we do not care because they will disappear soon anyway (`zapdisable` or 'tailcallstress' for example).

Reply:
By "disappear soon", I hope you mean, "will appear soon in AzDO runs"?

Reply:
I am not aware if such work item exists somewhere, do we have a GitHub issue to track that?

Reply:
I don't know. Is AzDO work (such as creating the full set of jobs that Jenkins previously ran) tracked in GitHub?